enterprise/backend/test/metabase_enterprise/serialization/v2/load_test.clj

```diff
@@ -702,8 +702,16 @@
           (ts/with-source-db
             (reset! user1s    (ts/create! User :first_name "Tom" :last_name "Scholz" :email "tom@bost.on"))
             (reset! user2s    (ts/create! User :first_name "Neil"  :last_name "Peart"   :email "neil@rush.yyz"))
```

Comment:
Adds in `:table_id` per [this](https://github.com/metabase/metabase/pull/34258/files#r1373208574) fix.